### PR TITLE
Added Jitsi Meet Integration

### LINKED ISSUE #353: Integrating Jitsi-Meet for Video Conferencing
Integrating Jitsi Meet into Codeuino using the JitsiMeetAPI.

### PR DESCRIPTION
**Pull request by theprover97**
Fixes #353 

I have created a basic integration of Jitsi Meet using JitsiMeetAPI.

### CODE REVIEW

**Comment by theprover97** [comment]
Jitsi1

**Comment by devesh-verma** [comment]
@theprover97 Few changes here:
1. Alignment of text, input fields and buttons is not proper.
2. Change text from `save` button to `join call`.
3. `Join call` button should not be clickable unless user has filled both the required fields.
4. instead of having `Type here..` as placeholder text change it to `Name` and `Room Id` respectively.

One question, how does user share this call with others?
can we have fullscreen option too.

**Comment by theprover97** [comment]
1. You can copy the share link from the chat window, which is simply "meet.jit.si/" followed by the room name. Sharing this with others allow multiple people to join the video chat.
2. To go full screen as well, an option is available within the chat window itself under settings.

**Comment by theprover97** [comment]
jitsi2

**Comment by theprover97** [comment]
@devesh-verma could you review these changes? Thanks.

File: src/user/integrations/NameForm.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    render(){      
+    return (
+      <Modal
+      {...this.props}
+      size="md"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered>
+      <div className="container">
+        <Modal.Header closeButton className="heading border border-0 p-0">
+        <Modal.Title id="contained-modal-title-vcenter">
```
Comment (devesh-verma) [inline src/user/integrations/NameForm.js]: please format the code properly, indentation is not proper.

**Comment by theprover97** [comment]
@devesh-verma I have made the changes.

**Comment by devesh-verma** [comment]
@theprover97 can you please squash the commits

**Comment by devesh-verma** [comment]
@theprover97 the can you resolve the conflict here.

**Comment by theprover97** [comment]
@devesh-verma I have resolved the merge conflicts.

**Pull request merged**